### DISCUSSION TITLE
Appium and ios native app.

### DISCUSSION THREAD

**Discussion by banditpig** [appium/appium#20062] [General]
Can Appium control an ios native app, written in Swift or Objective c, from a windows PC using net c#?
Thanks.

**Comment by KazuCocoa** [comment]
No, https://appium.github.io/appium-xcuitest-driver/latest/installation/requirements/ is the system requirement. It may have some methods to work with some of our toolchains, but the official one is only via macOS host.

**Reply by banditpig** [reply]
thanks
